test-two/src/index.js

```diff
@@ -3,21 +3,36 @@ import { getUsersForOrganisation } from "./services/github";
 
 const baseUrl = "https://api.github.com/";
 const organisationId = "google";
-const gitHubApiAuthToken = ""; // add your GitHub API OAuth key here to increase request limit
+const gitHubApiAuthToken = "";
+export const pages = 22; // There are 2144 users so 22 pages are sufficient
 
 export const go = async () => {
-  try {
+  let output = "";
+  let parallellCalls = [];
+
+  const getPageOfUsers = async pageNumber => {
```

Comment:
As far as I could tell, it was necessary to declare the getPageOfUsers function so that the following would work:

```
for (let i = 1; i <= pages; i++) {
  parallelCalls.push(getPageOfUsers(i));
}
```

This would then give me the array of promises that I use in the Promise.all() call at the bottom.

With that said, if I hadn't taken the Promise.all() approach, it wouldn't have been necessary to declare getPageOfUsers.

I can't really think of any potential downsides to it at the moment but I'm probably missing something since you mentioned it.